src/main/java/bio/terra/cli/service/utils/AwsUtils.java

```diff
@@ -0,0 +1,66 @@
+package bio.terra.cli.service.utils;
+
+import bio.terra.cli.exception.SystemException;
+import bio.terra.workspace.model.AwsCredential;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.net.URLConnection;
+import org.apache.http.client.utils.URIBuilder;
+import org.json.JSONObject;
+
+/** Utilities for AWS resources. */
+public class AwsUtils {
+  /**
```

Comment:
We also have a COW for this: https://github.com/DataBiosphere/terra-cloud-resource-lib/blob/master/src/main/java/bio/terra/cloudres/aws/console/ConsoleCow.java#L64